path/src/env.ps1

```
@@ -0,0 +1,40 @@
+﻿function script:Set-MissingVariablesForWindowsPowershell {
+	[System.Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidAssignmentToAutomaticVariable', '', Justification = 'all assignments to "automatic" variables are safe in this function')]
+	param()
+	if ($PSEdition -eq "Desktop") {
+		try { $global:IsWindows = $true } catch {}
+	}
+}
+Set-MissingVariablesForWindowsPowershell
+
+Start-Job -ScriptBlock {
+	param($FOUNT_DIR)
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'FOUNT_DIR' has been declared but not used.

(PSReviewUnusedParameter)